ppr-ui/tests/unit/RegistrationTable.spec.ts

```diff
@@ -278,9 +278,9 @@ describe('Test registration table with results', () => {
           startDate: null,
           status: ''
         })
-      }, 1000)
+      }, 3000)
```

Comment:
What's the reason for increase?

Reply:
I don't mind experimenting with different times, but I think 3 is a happy medium.  Different machines will give different results.  The main area the test is failing is locally... For example, when I changed to 1 second the test failed..  It sometimes fails with 3 seconds as well.

Reply:
hmm i see... testing UI with a 3s timeout goes beyond what unit test suppose to do, especially if it still fails sometimes (as you say). But I guess, let's monitor and adjust in the future if need be.